deploy-as-code/helm/environments/mgramseva-qa.yaml

```diff
@@ -231,6 +231,15 @@ ws-services:
 ifix-reference-adapter:
   keycloak-credentials-clientid: "mgramseva-qa"
 
+pspcl-ifix-adapter:
+  pspcl-fetch-bill-url: "https://billapp.pspcl.in//fetchbillingdata.asmx"
+  pspcl-fetch-payment-url: "https://billapp.pspcl.in//fetchbillingdata.asmx"
+  keycloak-pspcl-credentials-clientid: "pspcl-ifix-qa"
+  tenant-id: "pb"
```

Comment:
If this refers to the tenant id to be provided for ifix publish api call, please provide a better name and not just tenant-id.

Reply:
I am keeping the name as "fiscal-event-tenantId".